### PR TITLE
Remove old countries lists

### PR DESCRIPTION
**Pull request by kamil-tekiela**
These lists are not in use. I also don't fully understand why we had 3 lists. 

The include has been modified to return the list rather than include global variable. I find it easier to read.

### CODE REVIEW

File: include/countries.inc [hunk 1]
```diff
@@ -167,7 +167,7 @@ $COUNTRIES = array(
 'QAT' => 'Qatar',
 'KOR' => 'Republic of Korea',
 'MDA' => 'Republic of Moldova',
-'REU' => 'R�union',
```
[the remark this answers is not in the document]
Comment (cmb69) [reply include/countries.inc:170]: Is there are character encoding issue? Oh right, it is; the file was encoded as ISO-8859-1 (or something like that), and the `é` is the only character in this file where that matters.

**Comment by cmb69** [comment]
> I also don't fully understand why we had 3 lists.

Maybe that was related to the old mirror system; or it was just added in advance to cater to some future demands.

**Comment by kamil-tekiela** [comment]
The lists were used previously in now gone code, e.g. `ug.php` but I still don't know why we had 3 of them. Greping for it didn't bring up any usages anymore.

File: include/countries.inc (hunk 1 above)
[the remark this answers is not in the document]
Comment (kamil-tekiela) [reply include/countries.inc:170]: Neither my IDE nor GH is showing me this. Strange. It looks the same in diff.

[the remark this answers is not in the document]
Comment (cmb69) [reply include/countries.inc:170]: See https://patch-diff.githubusercontent.com/raw/php/web-php/pull/569.patch; the patch is UTF-8 encoded, and contains:
````diff
-'REU' => 'R�union',
+'REU' => 'Réunion',
````
And yes, that file should actually be UTF-8 encoded.

**Pull request merged**